index.html

```diff
@@ -12,6 +12,21 @@
 </head>
 
 <body>
+    <header>
+        <nav>
+            <img src="img/lambda-black.png" alt="lambda logo">
+            <a href="/web-Sprint-Challenge-User-Interface-and-Git/index.html">Home</a>
```

Comment:
You have an idea of how to make a multi-page website using HTML and CSS but have to clear up a few inconsistencies. In lines 18 and 19, you understand you need dynamically set the location of the Home and About-us page using the href attribute but you need to connect them relative to your file structure. You can do something like this:

```
<a href="index.html">home</a>
```

Reply:
Hi Josue, Thank You so much for your feedback man I really appreciate it. I understand what I need to do now to fix the href links locations